tests/hello/link.ld

```diff
@@ -20,7 +20,7 @@ MEMORY
 {
   ROM   (rx) : ORIGIN = 0x00000000, LENGTH = 32K
   ICCM (rwx) : ORIGIN = 0x40000000, LENGTH = 128K
-  DCCM (rwx) : ORIGIN = 0x50000000, LENGTH = 128K
+  DCCM (rwx) : ORIGIN = 0x40020000, LENGTH = 128K
```

Comment:
Do we need to update `DCCM`, `ICCM` to align to the change for using single block of RAM?

Reply:
Sure. This is a small example, but it's easy enough to switch to a single block.